python/fapolicy_analyzer/locale/fapolicy_analyzer.pot

```diff
@@ -7,9 +7,9 @@
 #, fuzzy
 msgid ""
 msgstr ""
-"Project-Id-Version: fapolicy-analyzer 0.0.4\n"
+"Project-Id-Version: fapolicy-analyzer snapshot\n"
```

Comment:
Wonder if this is being updated when an RPM build is performed...

Reply:
If that's the case, wouldn't we see the issue if we installed the newly created rpm on build system? I can try it then rollback the system.

Reply:
Will verify this in #148 

Non-blocking for this PR